Detectors/MUON/MID/Simulation/src/DigitsSorter.cxx

```diff
@@ -0,0 +1,65 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file   MID/Simulation/src/DigitsSorter.cxx
+/// \brief  Implementation of the digits Sorter for MID
```

Comment:
Do I understand correctly from the code that this class is not sorting but grouping digits ?
If that's the case we might want to change its name simply ?

Reply:
I will change it into DigitsPacker so that it reflects closely what it does.